R/run.R

```diff
@@ -63,7 +69,21 @@ CmdStanRun <- R6::R6Class(
         private$output_files_[ok]
       }
     },
-
+    profile_files = function(include_failed = FALSE) {
+      if (!any(file.exists(private$profile_files_))) {
+        stop(
+          "No profile files found. ",
+          "The model that produced the fit did not use any profiling.",
+          call. = FALSE
+        )
+      }
+      if (include_failed) {
+        private$profile_files_[file.exists(private$profile_files_)]
+      } else {
+        ok <- self$procs$is_finished() | self$procs$is_queued()
+        private$profile_files_[ok && file.exists(private$profile_files_)]
+      }
+    },
```

Comment:
@rok-cesnovar I think the tests are failing because now that `private$profile_files_` defaults to `NULL` it can't be passed to `file.exists()`. But I think you can just copy what we have for `latent_dynamics_files`, which doesn't use `file.exists()` anywhere.

Reply:
Thanks, I think that should do it yes.

Reply:
I think you might need to remove all uses of file.exists() not just the last one. Instead of 

```r
if (!any(file.exists(private$profile_files_)))
```

you can use 

```
if (!length(private$profile_files_))
```

and instead of 

```r
private$profile_files_[file.exists(private$profile_files_)]
```

maybe just 

```r
private$profile_files_
```

Reply:
Done!